README.md

```diff
@@ -1,6 +1,10 @@
 # @metamask/template-snap-monorepo
 
-This repository demonstrates how to develop a snap with TypeScript. For detailed instructions, see [the MetaMask documentation](https://docs.metamask.io/guide/snaps.html#serving-a-snap-to-your-local-environment).
+---
+
+## ⚠️ **NOTE**: Do not clone this repository. It is meant to be used through the [Create snap CLI](github.com/metaMask/snaps-monorepo/packages/create-snap).
```

Comment:
Not sure I agree. The nice thing about this repo is that you can create a new repository (not a fork) from it:

<img width="175" alt="image" src="https://user-images.githubusercontent.com/7503723/235376349-44db4681-dd26-4d68-915a-a72df0ba8a91.png">

Reply:
That's pretty much what the `mm-snap init` command does. We had discussed a while back that we want devs to use the CLI to create a new snap project, as this will allow us to expand the capabilities in the future. Just like with the Create React App CLI you can parametrize to create a typescript project by adding command line flags, we can allow e.g. `yarn create @metamask/snap --template=tx-insights`. The CLI already "creates a new repo" by nuking the Git directory and creating a new one.

Reply:
Wait what? I thought we were allowing both, as we already are? 🤔